### ISSUE TITLE
getNavMeshVertices not returning any vertices

### ISSUE THREAD

**Issue by GGAlanSmithee** [vincent/recast.js#22]
Trying to visually debug the navmesh, I attempted to duplicate what you (@vincent) did [here](https://github.com/vincent/recastjs-editor/blob/master/js/recastjs-editor/Sidebar.Navmesh.js#L214) in the recast.js editor.

However, `getNavMeshVertices` never returns any vertices. The method is not documented, so not sure if it's supported. If so, is there any other way to output the navmesh visually? I know it's possible to get the polygons using `queryPolygons`, but not quite sure how to draw them since there are no indecis.

Thanks in advance.

**Comment by vincent** [comment]
Yeah I remember I wasn't satisfied of the method, it's both buggy and performance hungry :(

How do you build the navmesh ? Tiled or Solo ?
Try both, maybe there is a bug in one.

`queryPolygons` should return the list of polys from one point to another, as a path.

**Comment by GGAlanSmithee** [comment]
@vincent thanks for the very quick answer!

I was using `buildTiled`. Using `buildSolo` does indeed work! Unfortunatley, I guess the two methods won't build the same navmesh?

**Comment by vincent** [comment]
Honestly I don't remember exactly, I guess they won't.

`getNavMeshVertice`s should probably be rewitten to handle tiles.

**Comment by GGAlanSmithee** [comment]
@vincent ok, thanks again! This is a great start, I will see what I can make of it. I might take a look at making `getNavMeshVertices` support tiles if I get some time over.
